src/test/java/org/junitpioneer/testkit/PioneerTestKitTests.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.testkit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class PioneerTestKitTests {
```

Comment:
But who tests our tests? (Is this necessary? I'm not sure...)

Reply:
well, good question actually - but i do think it is worth testing, we never know where we end up with this, and maybe people start using our PioneerTestKit in their extensions :D

Reply:
It's in the test root - do we ship it? I guess the code is open-source but I don't think `PioneerTestKit` is part of our "artifact"... or is it?

Reply:
good question

Reply:
> But who tests our tests? (Is this necessary? I'm not sure...)

Who watchs the watchers? JUnit Jupiter is tested by JUnit Jupiter.

> It's in the test root - do we ship it? I guess the code is open-source but I don't think `PioneerTestKit` is part of our "artifact"... or is it?

Content in `src/test` is not shiped.